src/keyboard/keyboard.ino

```diff
@@ -1,18 +1,45 @@
-// #include <usbhub.h>
-
 #include "NeoReportParser.h"
 
+
+class MouseRptParser : public MouseReportParser
+{
+protected:
+    void Parse(USBHID *hid, bool is_rpt_id, uint8_t len, uint8_t *buf);
+};
+
+void MouseRptParser::Parse(USBHID *hid, bool is_rpt_id, uint8_t len, uint8_t *buf)
+{
+  // Run parent class method.
+  MouseReportParser::Parse(hid, is_rpt_id, len, buf);
+  
+  if (len == 3) {
+    uint8_t mouseRpt[4];
+    mouseRpt[0] = buf[0];
+    mouseRpt[1] = buf[1];
+    mouseRpt[2] = buf[2];
+    mouseRpt[3] = 0;  // somehow this byte is missing with this lib
+    HID().SendReport(1,mouseRpt,sizeof(mouseRpt));
+  }
+}
+
 USB     Usb;
-//USBHub     Hub(&Usb);
-HIDBoot<USB_HID_PROTOCOL_KEYBOARD>    HidKeyboard(&Usb);
 
-NeoReportParser Prs;
+HIDBoot<USB_HID_PROTOCOL_KEYBOARD | USB_HID_PROTOCOL_MOUSE> HidComposite(&Usb);
```

Comment:
This composite initialization is very fragile. If you change a little do a test if everything still works.